application/config/internal.php

```diff
@@ -186,8 +186,9 @@
             'class' => defined('YII_DEBUG') && YII_DEBUG ? 'system.caching.CDummyCache' : 'CFileCache',
         ),
         // For more info about the emcache, see application/helpers/expressions/em_cache_helper.php.
+        // Disabled by default. Enable by adding emcache in config.php after installation.
```

Comment:
Arg … i'm not ready now … and previous default seems OK, no ?

Reply:
No, there are risks for bugs, so I don't want to enable it by default until we feel more sure. Also, don't waste time reviewing this until it's more finished. ;)